magma/magma-cuda110/cmakelists.patch

```diff
@@ -0,0 +1,332 @@
+--- CMakeLists.txt	2020-06-18 14:34:00.713576314 -0700
++++ CMakeLists.txt	2020-06-18 14:36:23.821607272 -0700
+@@ -3,7 +3,7 @@
+ # ----------------------------------------
+ # to disable Fortran, set this to "off"
+ # see also -DADD_ below
+-option( USE_FORTRAN "Fortran is required for some tester checks, but can be disabled with reduced functionality" ON )
++option( USE_FORTRAN "Fortran is required for some tester checks, but can be disabled with reduced functionality" OFF )
```

Comment:
Why this change is needed, if cmake is called with `-DUSE_FORTRAN=OFF`?

Reply:
I don't know, this part is copied from CUDA 10.2 patch.